assets/src/pages/room/components/StreamPlayer/simulcast/SimulcastEncodingToReceive.tsx

```diff
@@ -1,30 +1,63 @@
 import React, { FC } from "react";
 import { TrackEncoding } from "@jellyfish-dev/membrane-webrtc-js";
-import { isTrackEncoding } from "../../../../types";
+import { Tooltip } from "./Tooltip";
+import { LayerButton } from "./LayerButton";
 
 type Props = {
-  setDesiredEncoding: (encoding: TrackEncoding) => void;
-  desiredEncoding?: TrackEncoding;
+  setTargetEncoding: (encoding: TrackEncoding) => void;
+  targetEncoding: TrackEncoding | null;
   disabled?: boolean;
+  tileSizeEncoding: TrackEncoding | null;
+  enableSmartEncoding: boolean;
 };
 
-export const SimulcastEncodingToReceive: FC<Props> = ({ desiredEncoding, setDesiredEncoding, disabled }: Props) => {
+export const SimulcastEncodingToReceive: FC<Props> = ({
+  targetEncoding,
+  setTargetEncoding,
+  disabled,
+  tileSizeEncoding,
+  enableSmartEncoding,
+}: Props) => {
   return (
-    <div className="absolute bottom-0 right-0 z-50 bg-white p-2 text-sm text-gray-700 opacity-80 md:text-base">
-      <label>Encoding to receive</label>
-      <select
-        disabled={disabled}
-        value={desiredEncoding}
-        onChange={(event) => {
-          const value = event.target.value;
-          if (!isTrackEncoding(value)) return;
-          setDesiredEncoding(value);
-        }}
-      >
-        <option value="h">High</option>
-        <option value="m">Medium</option>
-        <option value="l">Low</option>
-      </select>
+    <div className="absolute bottom-0 right-0 z-50 w-full bg-white p-2 text-sm text-gray-700 opacity-80 md:text-base">
+      <div className="flex flex-row justify-between">
+        {enableSmartEncoding ? (
```

Comment:
I'm OK to separate the logic into two variants, but sometimes we wanted to force some layer in case something broke to investigate - thus it would be best to move the `enableSmartEncoding` switch from the join screen into the room itself